hypervisor/include/arch/x86/virtual_cr.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2018 Intel Corporation. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef VCR_H
+#define VCR_H
+
+/* CR0 bits hv want to trap to track status change */
+#define CR0_TRAP_MASK (CR0_PE | CR0_PG | CR0_WP | CR0_CD | CR0_NW)
+#define CR0_RESERVED_MASK ~(CR0_PG | CR0_CD | CR0_NW | CR0_AM | CR0_WP | \
+			   CR0_NE |  CR0_ET | CR0_TS | CR0_EM | CR0_MP | CR0_PE)
+
+/* CR4 bits hv want to trap to track status change */
+#define CR4_TRAP_MASK (CR4_PSE | CR4_PAE | CR4_VMXE | CR4_PCIDE)
+#define	CR4_RESERVED_MASK ~(CR4_VME | CR4_PVI | CR4_TSD | CR4_DE | CR4_PSE | \
+				CR4_PAE | CR4_MCE | CR4_PGE | CR4_PCE |     \
+				CR4_OSFXSR | CR4_PCIDE | CR4_OSXSAVE |       \
+				CR4_SMEP | CR4_FSGSBASE | CR4_VMXE |         \
+				CR4_OSXMMEXCPT | CR4_SMAP | CR4_PKE |        \
+				CR4_SMXE | CR4_UMIP)
+
```

Comment:
This 2 MACROs are only used by virtul_cr.c, should we move it to .c file?

Reply:
right, will check again. thanks!